src/Symfony/Bundle/FrameworkBundle/Tests/Functional/SerializerTest.php

```diff
@@ -67,6 +69,15 @@ public static function provideNormalizersAndEncodersWithDefaultContextOption():
             ['serializer.encoder.csv.alias'],
         ];
     }
+
+    public function testSerializeTranslatableBackedEnum()
+    {
+        static::bootKernel(['test_case' => 'Serializer']);
+
+        $serializer = static::getContainer()->get('serializer.alias');
+
+        $this->assertEquals(TranslatableBackedEnum::Get->value, $serializer->serialize(TranslatableBackedEnum::Get, 'yaml'));
```

Comment:
Nit-picking: Fetching the value from the enum feels like we're duplicating the logic of the normalizer. We should compare with a string literal instead.

```suggestion
        $this->assertEquals('GET', $serializer->serialize(TranslatableBackedEnum::Get, 'yaml'));
```

Reply:
I agree. We should be writing the value directly